src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFXSim.java

```diff
@@ -93,8 +96,11 @@ public SwerveModuleIOTalonFXSim(
         );
         this.turnSim.attachFeedbackSensor(new SimPhoenix6CANCoder(turnEncoder));
 
-        this.velocityVoltage = new VelocityVoltage(0);
+        this.velocityTorqueCurrentFOC = new VelocityTorqueCurrentFOC(0);
         this.positionVoltage = new PositionVoltage(0);
+        this.odometryThreadRunner = odometryThreadRunner;
+        this.odometryThreadRunner.registerControlRequest(driveMotor, velocityTorqueCurrentFOC, driveMotor::setControl);
+        this.odometryThreadRunner.registerControlRequest(turnMotor, positionVoltage, turnMotor::setControl);
```

Comment:
Were these calls were missing from the previous simulation implementation?

Reply:
yes